packages/itwinui-css/src/slider/slider.scss

```diff
@@ -38,6 +40,8 @@ $tick-height: var(--iui-size-s); // 12px
 
 @mixin iui-slider-rail {
   position: absolute;
+  width: 100%;
+  height: $track-height;
```

Comment:
It doesn't seem to me like the height and width need to be defined here if they're being defined under `@mixin iui-slider-horizontal > .iui-slider-rail` and `@mixin iui-slider-vertical > .iui-slider-rail`. Why did you add these here?

Reply:
**[DONE]** Ohh, right, thanks! These might have accedentally stayed in, but are not needed. Removed it now.